translation_service/src/serge_ts_connector.py

```diff
@@ -11,6 +11,12 @@
 utils_path = f'{root_path}/common/src'
 sys.path.append(utils_path)
 
+# Paths for each directory based on environment variables
+SERGE_TS = os.getenv('SERGE_TS')
+TS_SERGE_COPY = os.getenv('TS_SERGE_COPY')
+TS_INBOX = os.getenv('TS_INBOX')
```

Comment:
Rapi is using os.environ.get in his PR. Any preference between that vs. this?

Reply:
I think they are actually the same thing. The python docs show `os.getenv` (https://docs.python.org/3/library/os.html) as the method for this. Looking at stackoverflow (https://stackoverflow.com/questions/16924471/difference-between-os-getenv-and-os-environ-get), it might be that it's just a wrapper for `os.environ.get`. I think it should be equivalent to use either.